package.json

```diff
@@ -43,7 +43,6 @@
     "compression": "^1.7.4",
     "cross-env": "^7.0.3",
     "express": "^4.18.2",
-    "express-prometheus-middleware": "^1.2.0",
     "isbot": "^3.6.5",
     "morgan": "^1.10.0",
     "prom-client": "^14.1.1",
```

Comment:
Thanks for the PR. Since we opt not to use Prometheus, I am unsure if we need `prom-client`. Other than that, it looks good to me.

Reply:
Great catch, removed in a new commit.